src/sdk/http/http-client.ts

```diff
@@ -7,6 +7,8 @@ export interface IHttpClient {
 
 	get(path: string, params?: any): Promise<any>;
 
+	download(path: string, handler: any): Promise<any>;
```

Comment:
This should be `download(path: string, handler: (fileName: string, output: Readable)): Promise<any>;`